### PR TITLE
- QL support

### PR DESCRIPTION
**Pull request by malamili**
- WeaverSDK insert
- System database

### CODE REVIEW

File: src/auth/AuthCtrl.coffee [hunk 1]
```diff
@@ -163,7 +163,7 @@ bus.on('usersList', (req, res) ->
  http://localhost:9487/read?payload={"user":"phoenix","accessToken":"eyJhbGciOiJSUzI1NiJ9.eyIkaW50X3Blcm1zIjpbImRlbGV0ZV9hcHBsaWNhdGlvbiIsInJlYWRfYXBwbGljYXRpb24iLCJjcmVhdGVfcm9sZSIsImRlbGV0ZV9yb2xlIiwiY3JlYXRlX3Blcm1pc3Npb24iLCJjcmVhdGVfYXBwbGljYXRpb24iLCJkZWxldGVfZGlyZWN0b3J5IiwiZGVsZXRlX3Blcm1pc3Npb24iLCJjcmVhdGVfZGlyZWN0b3J5IiwicmVhZF91c2VyIiwiY3JlYXRlX3VzZXIiLCJyZWFkX3Blcm1pc3Npb24iLCJkZWxldGVfdXNlciIsInJlYWRfcm9sZSIsInJlYWRfZGlyZWN0b3J5Il0sInN1YiI6Im9yZy5wYWM0ai5tb25nby5wcm9maWxlLk1vbmdvUHJvZmlsZSNwaG9lbml4IiwiJGludF9yb2xlcyI6WyJwaG9lbml4Il0sIl9pZCI6IjU4NmY1OGNiYTFkMTQ3MTk0OTM2YTI2YSIsImV4cCI6MTQ4NDA0MDQwMSwiaWF0IjoxNDgzOTU0MDAxfQ.H7CEIbc_9157SHrU7VO0aBV9a40AAavPY4HEPm6Qw1AS0mrXWW5Ae4Sv-4lm0PwxvG1LhTsp9ZFW-faeNWhmJN0Aj37ZLLV5MEpFyVIpl91FnA_g0jKHWedRdzX8NvPcNMGqempWc49hgzLyFsm71Zcqp1ah2IaZ_oIHOGahz-DyUzkFI3hEF67iZeYrAfQp42a-Gi40QYUKOUPxbNBfAMQe5QcbyB1Qs75RwXg5AwJknGfyrfz4gNkkIEkAV5kvvoSoLFBsvi-v_NzkgQjh1DQbjim4X8dXIDEq7GX5b8OxEg6zrwKcdEQazdyYm1g8HgerivcdYOBB8Z9HKy3vAQ"}
 ###
 
-bus.filter('read', (req, res) ->
+bus.filter('readOff', (req, res) ->
```
Comment (gvdent) [inline src/auth/AuthCtrl.coffee]: I think this is a local development thing?

File: src/cli/funnies.coffee [hunk 2]
```diff
@@ -207,7 +207,12 @@ messages = [
   "Error: Cannot find module 'ifitaintbrokedontfixit'"
   "For more funnies, please donate by clicking on our Amazon banner."
   "I'll do later. I'm upgrading the coffee machine firmware."
+  "Are you ready? I was compiled ready!"
+  "Wings in working position, north sensors charged, eggs stable."
+  "More coffee! More coffee!"
+  "404, purpose of life not found."
+  "Permission to wipe database and retire."
```
Comment (gvdent) [inline src/cli/funnies.coffee]: These even make code reviewing better :D

File: src/cli/splash.coffee [hunk 3]
```diff
@@ -27,17 +27,18 @@ _ "HTTP:       #{BOOL conf.get('comm.http.enable')}"
 _ "Socket.io:  #{BOOL conf.get('comm.socket.enable')}"
 _ ""
 _ "# Services"
-_ "Database:   #{conf.get('services.database.endpoint')}"
-_ "Flock:      #{conf.get('services.flock.endpoint')}"
```
Comment (gvdent) [inline src/cli/splash.coffee]: Flock seems to be removed in some places, but not in all.. Removed here and in the docker compose, still present in custom-environment-variables.coffee. AuthCtrl still seems to require it however, so what is the flock status?

File: src/index.coffee [hunk 4]
```diff
@@ -34,12 +34,14 @@ require('app-module-path').addPath("src/#{path}") for path in [
...
-conf    = require('config')       # Configuration loads files in the root config directory
-Server  = require('server')
-splash  = require('splash')
-sounds  = require('sounds')
-pjson   = require('../package.json')
+Promise  = require('bluebird')
+conf     = require('config')       # Configuration loads files in the root config directory
+Server   = require('server')
+splash   = require('splash')
+sounds   = require('sounds')
```
Comment (bastbijl) [inline src/index.coffee]: sounds, why?

Comment (gvdent) [inline src/index.coffee]: Some people like them apparently ;) You can turn them off via the local config though.

File: src/cli/splash.coffee (hunk 3 above)
Comment (malamili) [inline src/cli/splash.coffee]: This was testing wip, should not have committed it. As Auth is off now anyways, i'll turn it back on after picking up User Management now

File: src/auth/AuthCtrl.coffee (hunk 1 above)
Comment (malamili) [inline src/auth/AuthCtrl.coffee]: Check!

File: src/cli/splash.coffee (hunk 3 above)
Comment (malamili) [inline src/cli/splash.coffee]: This was testing wip, should not have committed it. As Auth is off now anyways, i'll turn it back on after picking up User Management now

**Pull request merged**